src/include/rbd/cg_types.h

```diff
@@ -0,0 +1,25 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
These don't appear to be needed as part of the end-user librbd API.  Therefore, can we move these to "include/rbd_types.h"?  Also, all new header files need to be added to the Makefile*.am file under the noinst_HEADERS list.